tests/test_all_rules.py

```diff
@@ -1449,8 +1449,9 @@ def test_group_field_in_schemas(self):
                         self.fail(f"{self.rule_str(rule)} alert suppression field {fld} not \
                             found in ECS, Beats, or non-ecs schemas")
 
-    @unittest.skipIf(PACKAGE_STACK_VERSION < Version.parse("8.14.0"),
-                     "Test only applicable to 8.14+ stacks for eql non-sequence rule alert suppression feature.")
+    @unittest.skipIf(PACKAGE_STACK_VERSION < Version.parse("8.14.0") or  # noqa: W504
+                     PACKAGE_STACK_VERSION >= Version.parse("8.18.0"),  # noqa: W504
+                     "Test only applicable to 8.14 to 8.17 stacks for eql non-sequence rule alert suppression feature.")
```

Comment:
the comment contradicts the condition there, I think. The test will apply to all versions between `8.14` (including) and `8.18` (excluding), not to `8.14` and `8.17` only.

If you want to match just these too, maybe a stricter check is necessary

Reply:
Yes the intent is to run for all versions 8.14,8.15,8.16 and 8.17. Start from 8.18 as this feature is supported we can ignore the check. 
Will redo the comment